.github/workflows/enso4igv.yml

```diff
@@ -0,0 +1,34 @@
+# This workflow will build a Java project with Maven, and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Enso Language Support for IGV
+
+on:
+  push:
+    branches: [ develop ]
+  pull_request:
+    branches: [ develop]
```

Comment:
Does it mean that it will be a part of every PR? I imagine we won't change this plugin frequently. @mwu-tow can we do any better than that?

Reply:
I tried to restrict the trigger to project files in 342b43ef3d7c89205d2c1fb4d45ced77009282c0 - hopefully that's correct fix.